Makefile

```diff
@@ -481,7 +481,7 @@ build-ci:
 # - node-test-commit-linux-coverage: where the build and the tests need
 #   to be instrumented, see `coverage`.
 run-ci: build-ci
-	$(MAKE) test-ci
+	$(MAKE) test-ci -j1
```

Comment:
is it because of https://github.com/nodejs/node/issues/23255?

Reply:
No because of https://github.com/nodejs/node/issues/22006

Reply:
No, because of https://github.com/nodejs/node/issues/22006